.github/workflows/main.yml

```
@@ -309,7 +309,56 @@ jobs:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3.12.0
+
+      - name: Build and push Apache image
+        uses: docker/build-push-action@v6.18.0
+        with:
+          context: apache
+          push: ${{ env.PUSH_IMAGE }}
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          annotations: ${{ steps.meta.outputs.annotations }}
+          build-args: |
```

actionlint (1.7.10)

property "build-apache" is not defined in object type {}

(expression)